reqwest/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+# Copyright (C) 2020 O.S. Systems Sofware LTDA
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[package]
+name = "reqwest"
```

Comment:
This should be actix_web_with_reqwest

Reply:
I think "actix_reqwest" is enough to avoid ambiguity.